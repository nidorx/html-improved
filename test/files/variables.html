

<vars
    simple-var="simple value"
    objectVar="{name:'name value'}"
    arrayVar="['one','two','tree']"
    expressionVar="(new Date(1431009118430).toISOString())"
    numberVar="25"
    booleanVar="true"
    />

<ul>
    <li>
        !{simpleVar}
    </li>
    <li>
        !{objectVar.name}
    </li>
    <li>
        !{arrayVar[1]}
    </li>
    <li>
        !{expressionVar}
    </li>
    <li>
        !{numberVar}
    </li>
    <li>
        !{booleanVar}
    </li>
    <li>
        !{conditionalVarSim}
    </li>
    <li>
        !{conditionalVarNao}
    </li>
</ul>

<vars
    conditionalVarSim="((numberVar > 20)? 'sim' : 'não')"
    conditionalVarNao="((numberVar < 20)? 'sim' : 'não')"
    />
