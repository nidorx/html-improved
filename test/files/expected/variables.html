<ul>
    <li>
        simple value
    </li>
    <li>
        name value
    </li>
    <li>
        two
    </li>
    <li>
        2015-05-07T14:31:58.430Z
    </li>
    <li>
        25
    </li>
    <li>
        true
    </li>
    <li>
        sim
    </li>
    <li>
        n√£o
    </li>
</ul>